app/services/doc_auth/errors.rb

```diff
@@ -120,7 +120,7 @@ module Errors
       GENERAL_ERROR => { long_msg: GENERAL_ERROR, field_msg: FALLBACK_FIELD_LEVEL, hints: true },
       # TODO, theses messages need modifying
       # Liveness, use general error for now
```

Comment:
Wonder if we should remove this comment
```suggestion
```

Reply:
[Done](https://github.com/18F/identity-idp/pull/10042/commits/3735fa6db22b2f787c160d4e76b924f85b6260d6).